third-party/stb_image.h

```diff
@@ -1644,13 +1644,13 @@ stbi_inline static int stbi__extend_receive(stbi__jpeg *j, int n)
    int sgn;
    if (j->code_bits < n) stbi__grow_buffer_unsafe(j);
 
-   sgn = (stbi__int32)j->code_buffer >> 31; // sign bit is always in MSB
+   sgn = j->code_buffer >> 31; // sign bit always in MSB; 0 if MSB clear (positive), 1 if MSB set (negative)
```

Comment:
Assumes int is 32 bits. Cast to 32 bits or use sizeof( int ) in the shift amount.

Reply:
Let priorities updating this third party as stated before.
If we cannot we will take the fix